src/routes/login/+page.svelte

```diff
@@ -29,48 +122,60 @@
                 <span>Email</span>
             </label>
             <input
-                class="input {emailIndicator}"
+                class="bg-surface-700 text-white border rounded-full transition-colors {emailIndicator} w-full focus:border-primary-500 focus:outline-none focus:ring-0"
                 type="text"
                 id="email_input"
                 placeholder="beispiel.organisation@mail.com"
+                bind:this={emailReference}
             />
+            {#if emailIndicator == indicatorStatus.warning}
+                <div
+                    class="flex flex-col justify-center items-center transition-opacity"
+                >
+                    <p class="text-sm text-warning-500">
+                        Bitte gib eine valide Email-Adresse ein
+                    </p>
+                </div>
+            {/if}
             <!-- Password field -->
             <label class="label" for="password_input">
                 <span>Passwort</span>
             </label>
-            <div
-                class="input-group input-group-divider relative grid-cols-[auto_1fr_auto]"
-            >
+            <div class="relative">
                 <input
-                    class="w-72 {passwordIndicator}"
+                    class="bg-surface-700 text-white border rounded-full transition-colors {passwordIndicator} w-full focus:border-primary-500 focus:outline-none focus:ring-0"
                     type={showPassword ? "text" : "password"}
                     id="password_input"
                     placeholder="Dein super sicheres Passwort 😉"
+                    bind:this={passwordReference}
                 />
-                <button
-                    class="variant-filled-secondary right-0 leading-5"
-                    on:click={() => (showPassword = !showPassword)}
-                >
-                    {#if showPassword}
-                        {btnIcon.unlocked}
-                    {:else}
-                        {btnIcon.locked}
-                    {/if}
-                </button>
+                <div>
+                    <button
+                        class=" absolute bg-surface-700 w-14 border-surface-500 inset-y-0 h-8 top-1 right-1 border-l-2 border-surface-400 pl-4 flex items-center text-md rounded-r-full"
```

Comment:
Hier bekomme ich eine warning, dass `border-surface-500` und `border-surface-400` die gleiche property setzen. Kann die Tailwind VSCode extension empfehlen, falls nicht bereits installiert.
```suggestion
                        class="absolute bg-surface-700 w-14 border-surface-500 inset-y-0 h-8 top-1 right-1 border-l-2 pl-4 flex items-center text-md rounded-r-full"
```